perf/benchmark/runner/runner.py

```diff
@@ -170,14 +171,19 @@ def run(self, conn, qps, size, duration):
         if self.mode == "grpc":
             grpc = "-grpc -ping"
 
+        cacert_path = ""
+        if self.cacert is not None:
+            cacert_path = "-cacert {cacert_path}".format(cacert_path=self.cacert)
+
         fortio_cmd = (
-            "fortio load -c {conn} -qps {qps} -t {duration}s -a -r {r} {grpc} -httpbufferkb=128 " +
+            "fortio load -c {conn} -qps {qps} -t {duration}s -a -r {r} {cacert_path} {grpc} -httpbufferkb=128 " +
```

Comment:
I think it would be much more clearer to put `-cacert` in this fortio load command instead of add it in line 176 `cacert_path`. Since `cacert_path` should be your path to the cacert.